tests/handlers/test_widgets.py

```diff
@@ -63,4 +63,4 @@ def test_delete_widget(self):
 
         self.assertEqual(rv.status_code, 200)
         dashboard = models.Dashboard.get_by_slug_and_org(widget.dashboard.slug, widget.dashboard.org)
-        self.assertEqual(dashboard.widgets.count(), 0)
+        self.assertEqual(len(dashboard.widgets), 0)
```

Comment:
It has been changed from `count()` to `len()`

Reply:
will be changing here, as dashboard object already contains widget objects, so len() function is applicable here

Reply:
OK. Understood.